test/e2e/kubeproxy.go

```diff
@@ -420,10 +425,10 @@ func (config *KubeProxyTestConfig) setup() {
 	By("Getting ssh-able hosts")
 	hosts, err := NodeSSHHosts(config.f.Client)
 	Expect(err).NotTo(HaveOccurred())
-	if len(hosts) == 0 {
```

Comment:
We won't run the test and the test will appear to be green when it was in-fact not run.
So, this check basically says that: something is wrong.